docs/tutorials/emulation/02_dim_reduction.ipynb

```diff
@@ -149,24 +149,39 @@
    "source": [
     "from autoemulate.core.compare import AutoEmulate\n",
     "from autoemulate.transforms import *\n",
+    "from torch.distributions.transforms import ReshapeTransform\n",
     "import torch \n",
     "\n",
     "# Convert to tensors\n",
-    "x, y = torch.Tensor(X).float(), torch.Tensor(Y).float()\n",
+    "x, y = torch.Tensor(X).float(), torch.Tensor(Y).float().reshape(-1, n, n)\n",
```

Comment:
Why is the y data being reshaped here as well as in the AutoEmulate class with the ReshapeTransform?

Reply:
The data is stored on disk / output from simulation currently flattened so this step is reshaping the ground truth back to the original shape.